mern-client/src/pages/shop/Shop.jsx

```
@@ -1,49 +1,49 @@
+
+        getBooks();
+    }, []);
+
+    return (
+        <div className="mt-10 px-4 lg:px-24">
+            <h2 className="text-5xl font-bold text-center">
+                All Books are here
+            </h2>
+            <div className="my-12 grid  gap-8 xl:grid-cols-4 sm:grid-cols-2 md:grid-cols-3 grid-cols-1">
+                {books.map((book) => (
+                    <Card className="">
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)